common/src/main/java/com/linecorp/centraldogma/common/PathPattern.java

```diff
@@ -29,8 +29,8 @@
  *   <li>{@code "*.json"} - all JSON files</li>
  *   <li>{@code "/foo/*.json"} - all JSON files under the directory {@code /foo}</li>
  *   <li><code>"/&#42;/foo.txt"</code> - all files named {@code foo.txt} at the second depth level</li>
- *   <li>{@code "*.json,/bar/*.txt"} - use comma to specify more than one pattern. A file will be matched
- *                                     if <em>any</em> pattern matches.</li>
+ *   <li>{@code "*.json","/bar/*.txt"} - if you have more than 1 pattern you can supply them as {@code varargs} or {@link Iterable}. 
```

Comment:
I prefer that we spell out 1

```suggestion
 *   <li>{@code "*.json","/bar/*.txt"} - if you have more than one pattern you can supply them as {@code varargs} or {@link Iterable}. 
```